suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -458,3 +458,9 @@ export const selectIsDeviceNotEmpty = createMemoizedSelector(
         return isNotEmpty;
     },
 );
+
+export const selectStakingAccounts = createMemoizedSelector([selectAccountByKey], account => {
+    if (!account) return null;
+
+    return account.stakingAccounts ?? null;
```

Comment:
why to return null instead of empty array?

Reply:
fixed https://github.com/trezor/trezor-suite/pull/15409/commits/45e2808344e49c757330ec18a4066252f1d98fc1